src/infra/auction-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+data:
```

Comment:
@amerkul structure your ConfigMap

`kind` goes first, after `apiVersion`
and only then your data